common/source/docs/common-tarot-gimbal.rst

```diff
@@ -96,8 +94,7 @@ change the configuration you should:
 #. In the configuration UI, select the COM port and press the **Open COM Port** button.  After a few seconds the image of the camera should begin moving as the gimbal is moved.
 #. The most important setting is to set the **Default Mode** (just above COM port drop-down) to **Default Mode: stick position mode**.  Select this and press **Write Settings To Flash**.  Alternatively to load all the 3DR recommended configuration:
 
-   #. First right-click on `this link to the 3DR_Tarot.bgsc file <https://raw.githubusercontent.com/diydrones/ardupilot/master/Tools/Frame_params/3DR_Tarot.bgsc>`__
-      and save to somewhere on your computer.
+   #. First right-click on `this link to the 3DR_Tarot.bgsc file <https://raw.githubusercontent.com/diydrones/ardupilot/master/Tools/Frame_params/3DR_Tarot.bgsc>`__ and save to somewhere on your computer.
```

Comment:
I can't find this file - not in https://github.com/ArduPilot/ardupilot/tree/master/Tools/Frame_params as expected. Thoughts?